openedx_external_enrollments/external_enrollments/pathstream_external_enrollment.py

```diff
@@ -35,29 +40,48 @@ def _post_enrollment(self, data, course_settings=None):
         """
         LOG.info('Calling enrollment for [%s] with data: %s', self.__str__(), data)
         LOG.info('Calling enrollment for [%s] with course settings: %s', self.__str__(), course_settings)
-
+        log_details = {
+            'data': str(data),
+            'course_advanced_settings': course_settings,
+        }
         try:
             enrollment, created = ExternalEnrollment.objects.get_or_create(  # pylint: disable=no-member
                 controller_name=str(self),
                 course_shell_id=data.get('course_id'),
                 email=data.get('user_email'),
             )
         except IntegrityError:
-            LOG.error('Failed to complete enrollment, course_id and user_email can\'t be None')
+            error_msg = 'Failed to complete enrollment, course_id and user_email can\'t be None.'
+            log_details['status'] = error_msg
+            LOG.error(error_msg)
```

Comment:
@Jacatove, would you add a blank line before this one, please?

Reply:
@Jacatove, I meant before not after. To separate assignment to action statements.